shared/kubernetes/provider.go

```diff
@@ -278,6 +278,36 @@ func (provider *Provider) GetMizuApiServerPodObject(opts *ApiServerOptions, moun
 				},
 			},
 		},
+		{
+			Name:            "basenine",
+			Image:           "ghcr.io/up9inc/basenine:v0.2.26",
+			ImagePullPolicy: opts.ImagePullPolicy,
+			VolumeMounts:    volumeMounts,
+			ReadinessProbe: &core.Probe{
+				FailureThreshold: 3,
+				Handler: core.Handler{
+					TCPSocket: &core.TCPSocketAction{
+						Port: intstr.FromInt(9099),
```

Comment:
this is also look like a magic number - and I don't even see that we pass it to the command so I guess this is the default value inside basenine.. maybe it can be better to specify this port in the command and make it variable that we pass to both places (here in the ReadinessProbe and down in the Args for the container command)

Reply:
@gadotroee done with https://github.com/up9inc/mizu/commit/b44f304618cd52cbf0eace292366ea79b3da46a4